src/gui/PrintService.cc

```diff
@@ -58,7 +58,7 @@ createPrintServices() {
   // TODO: Where to call this, will we need a mutex?
   try {
     auto networkRequest = NetworkRequest<void>{
-        QUrl{"http://localhost:8000/print-service.json"}, {200}, 30};
+        QUrl{"https://files.openscad.org/print-service.json"}, {200}, 30};
```

Comment:
Just an idea, should we give it a dedicated alias name? In case we need/want to move it to a different location?

Reply:
Good idea, any suggestion for URL? app.openscad.org? We need SSL for that domain though, but that should be easy to set up using the existing SSL stuff.

Reply:
Multi-domain handling for SSL is fully automatic via https://github.com/nginx-proxy/acme-companion. It's trivial to add either a dedicated nginx container or just point this to files.openscad.org and let it generate a certificate for both domains. Well, I'm sure about the first solution, 2nd according to documentation :grin:.

Reply:
`app.openscad.org` is live as alias for `files.openscad.org`.

Reply:
awesome!
I guess I should switch the macOS autoupdater as well